components-api/src/main/scala/pl/touk/nussknacker/engine/api/context/ContextTransformation.scala

```diff
@@ -105,6 +108,9 @@ object ContextTransformation {
     })
 
   class DefinedByBuilder(definition: ContextTransformationDef) {
+    def notImplemented[T: NotNothing: ClassTag]: ContextTransformation =
```

Comment:
this is a part of public api - mby lets add some note that it is for testing purposes of typing/scenario validation?